testing/proftpd/proftpd.initd

```diff
@@ -0,0 +1,28 @@
+#!/sbin/openrc-run
+
+pidfile="/run/proftpd/proftpd.pid"
+command="/usr/sbin/proftpd"
+command_args="-n -q"
+command_background="yes"
+start_stop_daemon_args="--quiet"
+required_files="/etc/proftpd/proftpd.conf"
+extra_started_commands="reload"
+
+depend() {
+	need net
+	use logger dns
+	after firewall
+}
+
+start_pre() {
+	ebegin
+	checkpath --directory ${pidfile%/*}
+	$command -t &>/dev/null
```

Comment:
`&>` is a bashism, not POSIX-sh syntax. Please replace it with `>/dev/null 2>&1`.